lib/reloadButton.js

```diff
@@ -27,9 +26,14 @@ module.exports = function refreshButton (_options) {
       '}')
   }
 
+  var existingLinks = document.querySelectorAll('a.' + options['class'])
+  ;[].forEach.call(existingLinks, function (link) {
+    document.body.removeChild(link)
+  })
+
   var link = document.createElement('a')
   link.className = options['class']
-  link.href = href
+  link.href = window.location.href
```

Comment:
this bit was so that we always reload from the URL the page originally loaded from, not when you call `reloadButton()` can make a difference when you have history API routes, etc. Can't think why you'd call `reloadButton()` after you've changed routes mind you, but this makes it pretty bullet proof.